tmt/base.py

```diff
@@ -2523,7 +2523,7 @@ def __init__(
             **kwargs)
         self._update_metadata()
 
-    # Override the parent implementation - it would try to call `Tree.storys()`...
+    # Override the parent implementation - it would try to call `Tree.stories()`...
```

Comment:
Not a typo, it's wrong on purpose, to demonstrate why the custom `from_tree` implementation is needed. The default would try to call `Tree.storys()`, and we need to avoid that.

Reply:
Ok, it seems I do need to add an `exclude-file` for something like that